### ISSUE TITLE
Some ByteStringSerializer's don't have a corresponding ByteStringDeserializer

### ISSUE THREAD

**Issue by francisdb** [etaty/rediscala#135]
Is there a reason that for example ByteStringDeserializer[Long] is missing?

**Comment by etaty** [comment]
Yes, because you can't write a ByteStringDeserializer[Long] for any ByteString.

**Comment by francisdb** [comment]
So why do you have a ByteStringDeserializer[Double] then, that one also does not work for any ByteString?

**Comment by etaty** [comment]
Hum I see, you are right.
DoubleRedis should/could be hidden? (It should be hidden as it is Redis specific)
https://github.com/etaty/rediscala/blob/master/src/main/scala/redis/Converter.scala#L183-L193

**Comment by francisdb** [comment]
Honestly, I think you should just have deserialisers for all of the serialisers. Even if they might throw exceptions. Users can always opt for the String version if they want to be safe.
Are you sure the RedisDouble is Redis specific? As far as I know there is no such thing as double in Redis. It's just parsing the jave Double#toString() output.
